test/system/assignments_test.rb

```diff
@@ -29,8 +29,8 @@ class AssignmentsTest < ApplicationSystemTestCase
   test 'assign unassigned client - client side' do
     visit clients_path
     first(:link, 'Need accompanying').click
-    click_link 'Find volunteer'
-    click_link 'Reserve'
+    page.find('a', text: 'Find volunteer').trigger('click')
+    page.find('a', text: 'Reserve').trigger('click')
```

Comment:
This should do the same as `click_link`

Reply:
Yes, but it failed once (only) in Travis like the other flappy test and I changed that in case we have the same problem.